chrome-launcher/chrome-launcher.ts

```diff
@@ -51,17 +51,24 @@ export async function launch(opts: Options = {}): Promise<LaunchedChrome> {
 
   const instance = new Launcher(opts);
 
+  let sigintListener =
+      async () => {
+    await instance.kill(sigintListener);
+    process.exit(_SIGINT_EXIT_CODE);
+  }
+
   // Kill spawned Chrome process in case of ctrl-C.
   if (opts.handleSIGINT) {
-    process.on(_SIGINT, async () => {
-      await instance.kill();
-      process.exit(_SIGINT_EXIT_CODE);
-    });
+    process.on(_SIGINT, sigintListener);
   }
 
   await instance.launch();
 
-  return {pid: instance.pid!, port: instance.port!, kill: async () => instance.kill()};
+  return {
+    pid: instance.pid!,
+    port: instance.port!,
+    kill: async () => instance.kill(sigintListener)
```

Comment:
instead of passing the sigint listener in here, how about we move the `if (opts.handleSIGINT)` to the Launcher instead?

that would keep this function cleaner and allow you to save the listener for removal to `this` instead